.github/workflows/plugin_install.yml

```diff
@@ -63,7 +62,6 @@ jobs:
           admin-password: ${{ steps.random-password.outputs.generated_name }}
 
       - name: Run sanity tests
-        uses: gradle/gradle-build-action@v2
+        uses: burrunan/gradle-cache-action@v1
         with:
-          cache-disabled: true
-          arguments: integTestRemote -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="opensearch" -Dhttps=true -Duser=admin -Dpassword=${{ steps.random-password.outputs.generated_name }} -i
+          arguments: integTestRemote -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="opensearch" -Dhttps=true -Duser=admin -Dpassword=${{ steps.random-password.outputs.generated_name }}
```

Comment:
Was the `-i` intentional here? My understanding is that it was not, and hence i removed it to prevent from printing extra logs, but if it was then I can revert this.

Reply:
Will the log still be shown after the removal of hte `-i`?

Reply:
`-i` enables printing info logs. Unless we absolutely need them, these will just use up extra network and memory for each run, and so there is an opportunity to save that by not adding -i